### DISCUSSION TITLE
Cell identifier of a Multi Scale tessellation

### DISCUSSION THREAD

**Discussion by Hx83866** [neperfepx/neper#154] [Q&A] [answered]
Hello Everyone,

at the moment I am writing my thesis, which requires simulation on a multi-scale RVE. I have successfully generated a multi-scale RVE (see in following figure).

![sheet_04](https://user-images.githubusercontent.com/35709665/112124997-41937b00-8bc3-11eb-8335-587a818a61c4.png)

As you can see from the figure, a 2.5D RVE with three hierarchical layers has been created and the colors identify the second layers. 

However, my problem is: 
**Is there any method that I can export the identifier of each cell in following pattern:** 
_Identifier_Lay_1 :: Identifer_Lay_2 :: cell identifier(Lay_3)_ 
**It means, through the exported information I could know which cell belongs to which Lay1 and Lay2**

From the manual (on Page 97 [B.3 Multiscale Cell File]) I knew it is possible to given the such information as one of the input files.  However, is it also possible to export the such file or anything similar  while tessellation?

Looking forward for your reply!

Thanks a lot!

RWTH Aachen Uni.
Xiang Hu

**Comment by rquey** [comment, marked as answer]
Yep, this would be `-statcell "scaleid(1),scaleid(2),scaleid(3)"`.
